it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html

```diff
@@ -0,0 +1,43 @@
+<div class="custom-embed">
+        <script>
+            
+           function loadform(btn) {
+             var formPath = btn.getAttribute('data-form-path')
+             loadAdaptiveForm({
+               formPath: formPath,
+               CSS_Selector: '.customafsection'
+             })
+           }
+
+           var loadAdaptiveForm = function(options) {
+             var path = options.formPath + '.html?wcmmode=disabled';
+             fetch(path).then((res) => res.text()).then(html => {
+               setInnerHTML(document.querySelector(options.CSS_Selector), html); // OR $(options.CSS_Selector).html(html);
+             })
+           }
+
+            function setInnerHTML(elm, html) {
+                elm.innerHTML = html;
+                Array.from(elm.querySelectorAll("script"))
```

Comment:
Why do you need to set script tag individually ? innerHTML should do it

Reply:
script set through innerHTML does not execute automatically.
https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#:~:text=in%20this%20case-,Although%20this,-may%20look%20like
We can either use jquery or if jquery is not present then the given method is one of the ways to include scripts in embedded html.